### ISSUE TITLE
Low Priority WCAG Level  AAA

### ISSUE THREAD

**Issue by markrowen** [digital-land/digital-land#721]
Areas of the service which fail to meet the WCAG 2.2 AAA requirements are not in scope for 
the purposes of this audit, however, where issues were encountered by the analysts, these 
have been reported. It's recommend that these issues are resolved.
